tools/src/main/java/org/apache/kafka/tools/consumer/ConsoleShareConsumerOptions.java

```diff
@@ -0,0 +1,268 @@
+                                " print.timestamp=true|false\n" +
+                                " print.key=true|false\n" +
+                                " print.offset=true|false\n" +
+                                " print.delivery=true|false\n" +
+                                " print.partition=true|false\n" +
+                                " print.headers=true|false\n" +
+                                " print.value=true|false\n" +
+                                " key.separator=<key.separator>\n" +
+                                " line.separator=<line.separator>\n" +
+                                " headers.separator=<line.separator>\n" +
+                                " null.literal=<null.literal>\n" +
+                                " key.deserializer=<key.deserializer>\n" +
+                                " value.deserializer=<value.deserializer>\n" +
+                                " header.deserializer=<header.deserializer>\n" +
+                                "\nUsers can also pass in customized properties for their formatter; more specifically, users can pass in properties keyed with 'key.deserializer.', 'value.deserializer.' and 'headers.deserializer.' prefixes to configure their deserializers.")
+                .withRequiredArg()
+                .describedAs("prop")
+                .ofType(String.class);
+        messageFormatterConfigOpt = parser.accepts("formatter-config", "Config properties file to initialize the message formatter. Note that " + messageFormatterArgOpt + " takes precedence over this config.")
+                .withRequiredArg()
+                .describedAs("config file")
+                .ofType(String.class);
+        maxMessagesOpt = parser.accepts("max-messages", "The maximum number of messages to consume before exiting. If not set, consumption is continual.")
+                .withRequiredArg()
+                .describedAs("num_messages")
+                .ofType(Integer.class);
+        timeoutMsOpt = parser.accepts("timeout-ms", "If specified, exit if no message is available for consumption for the specified interval.")
+                .withRequiredArg()
+                .describedAs("timeout_ms")
+                .ofType(Integer.class);
+        rejectOpt = parser.accepts("reject", "If specified, messages are rejected as they are consumed.");
+        releaseOpt = parser.accepts("release", "If specified, messages are released as they are consumed.");
+        rejectMessageOnErrorOpt = parser.accepts("reject-message-on-error", "If there is an error when processing a message, " +
+                "reject it instead of halt.");
+        bootstrapServerOpt = parser.accepts("bootstrap-server", "REQUIRED: The server(s) to connect to.")
+                .withRequiredArg()
+                .describedAs("server to connect to")
+                .ofType(String.class);
+        keyDeserializerOpt = parser.accepts("key-deserializer")
```

Comment:
This needs a parameter description: "The name of the class to use for deserializing keys.".